code/modules/clothing/gloves/color.dm

```diff
@@ -165,7 +165,37 @@
 	item_state = "white"
 	_color = "mime"
 	species_fit = list(VOX_SHAPED)
-
+	
+/obj/item/clothing/gloves/white/advanced //mime traitor gloves, spawn in a silent hand gun with two shots 
+	action_button_name = "Prime Gun"
+	var/obj/item/weapon/gun/projectile/handgun/current_gun = null
+	var/charging = FALSE
+	
+/obj/item/clothing/gloves/white/advanced/attack_self(mob/user)
+	var/mob/living/carbon/human/M = user
+	if(!istype(M))
+		return
+	if(current_gun)
+		to_chat(M, "<span class ='notice'>Your gun evaporates into thin air!</span>")
+		qdel(current_gun)
+		current_gun = null
+	if(!charging)
+		if(!M.get_active_hand())
+			var/obj/item/weapon/gun/projectile/handgun/G = new
+			current_gun = G
+			if(!M.miming) //nonmimes get a loud version
+				G.silenced = FALSE
+				G.fire_sound = 'sound/weapons/Gunshot.ogg'
+			M.put_in_active_hand(G)
+			to_chat(M, "<span class ='notice'>You begin to channel an invisible gun through your fingers!</span>")
+			charging = TRUE
+			spawn(50)
```

Comment:
Since this takes 5 seconds and starts charging from the moment I summon the gun, why would I not prep a gun, wait a bit, fire twice, desummon, resummon, and fire twice again? That's insta-crit.